### PR TITLE
Announcements

### LINKED ISSUE #804: Banco de Editais
## Feature Description

Do mesmo molde do Banco de Ideias o banco de editais será possível visualizar os variados editais que são publicados por meio de outras instituições. Queremos dar a oportunidade para os nossos usuários fiquem sabendo quais editais vigentes eles podem consultar e participar.

Essa funcionalidade terá:

1 - LISTA DE EDITAIS

Listar todos os editais cadastrados em formato de card (similar ao banco de ideias)

O conteúdo dessa lista será:

Organização:
Número do Edital:
Título:
Descritivo:
Público-Alvo:
Palavras-chave:
Recursos financeiros:
Período de inscrição:
Observação:
Link para a Página do edital: (abrir em uma nova janela)
Somente serão publicos os registros com status PUBLISHED

Mesmo o edital com data expirada continua visível nessa lista

2 - REGISTRAR UM EDITAL

Qualquer usuário cadastrado poderá registrar um edital.

Os campos serão:

Organização (Dropdown):
Número do Edital: (campo texto)
Título: (campo texto)
Descritivo: (textarea)
Público-Alvo: (Lista de publico-alvo das taxonomias existentes)
Palavras-chave: (Lista de palavras-chave das taxonomias existentes)
Recursos financeiros: (Opcional) - campo de valor monetário
Período de inscrição: (data de inicio e data de fim)
Observação: (textarea)
Link para a Página do edital: (URL)
Ao cadastrar o registro terá um status de pendente

Somente o ADMIN muda o status para PUBLISHED

O ADMIN poderá cancelar o registro.

---------------

## Acceptance criteria

* 

## Implementation Brief

*

### PR DESCRIPTION
**Pull request by luizeboli**
## Summary

Resolves #804 

https://www.loom.com/share/95e3e9254eec412bace0a69286757a47

## QA Steps

1. Access homepage
2. Click "Recursos > Banco de editais" in Header
3. Verify that you can interact with cards and buttons
4. Try to register a new announcement and validate all fields

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/web/components/Button/styles.js [hunk 1]
```diff
@@ -208,6 +210,9 @@ export const RectangularButton = styled.button`
 		text-transform: uppercase;
 		text-align: center;
 
+		margin: ${!!autoX && '0 auto'};
```
Comment (mateus4k) [inline packages/web/components/Button/styles.js:213]: 💯

File: packages/web/components/Form/SelectField.js [hunk 2]
```diff
@@ -162,6 +162,11 @@ const SelectField = ({
 				name,
 				selectedValue.map((value) => options.find((option) => option.value === value)),
 			);
+		} else if (typeof selectedValue === 'object') {
+			setValue(
+				name,
+				options.find((option) => option.value === selectedValue.value),
+			);
```
Comment (mateus4k) [inline packages/web/components/Form/SelectField.js:169]: Great trick! 🚀

Comment (luizeboli) [reply packages/web/components/Form/SelectField.js:169]: No more bugs haha

File: packages/web/components/LandingPage/Button/index.js [hunk 3]
```diff
@@ -74,16 +72,16 @@ const Button = ({
 	}
 
 	return isLink ? (
-		<Link bgColor={bgColor} color={colors.white} {...rest}>
+		<Link $bgColor={bgColor} $color={colors.white} {...rest}>
```
Comment (luizeboli) [inline packages/web/components/LandingPage/Button/index.js:75]: You'll like that one @mateus4k  😆

**Review by mateus4k — APPROVED** [review]
Good job 🚀

**Pull request merged**